modules/api/test/LorisApiInstrumentsTest.php

```diff
@@ -0,0 +1,644 @@
+<?php
+
+require_once __DIR__ . "/LorisApiAuthenticatedTest.php";
+
+/**
+ * PHPUnit class for API test suite. This script sends HTTP request to every enpoints
+ * of the api module and look at the response content, status code and headers where
+ * it applies. All endpoints are accessible at <host>/api/<version>/
+ * (e.g. the endpoint of the version 0.0.3 ofd the API "/projects" URI for the host
+ * "example.loris.ca" would be https://example.loris.ca/api/v0.0.3/projects)
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Integration
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class LorisApiInstrumentsTest extends LorisApiAuthenticatedTest
+{
+    protected $instrumentTest = "medical_history";
+    protected $candidTest     = "300004";
+    protected $visitTest      = "V1";
```

Comment:
looking at https://github.com/aces/Loris/blob/main/raisinbread/RB_files/RB_session.sql#L937 , I think only V3 has medical_history for that candidate.